persistence/gorm/gorm.go

```diff
@@ -87,15 +93,17 @@
 	}
 
 	// After successful DB initialization, migrate the schema
-	// Migrate User
-	if err = g.db.AutoMigrate(&auth.User{}); err != nil {
-		err = fmt.Errorf("error during auto-migration: %w", err)
-		return
+	var types = []interface{}{
+		&auth.User{},
+		&orchestrator.CloudService{},
+		&assessment.MetricImplementation{},
 	}
-	// Migrate CloudService
-	if err = g.db.AutoMigrate(&orchestrator.CloudService{}); err != nil {
-		err = fmt.Errorf("error during auto-migration: %w", err)
-		return
+
+	for _, typ := range types {
```

Comment:
```suggestion
	for _, type := range types {
```

Reply:
`type` is a reserved keyword, that was the reasoning behind `typ`.